### PR TITLE
Feature/eslint extend

### PR DESCRIPTION
**Pull request by Kaskyi**
Add additional, eslint-scss rules.

### CODE REVIEW

File: src/app/account/social/social-register.component.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export class SocialRegisterComponent implements OnInit {
 
     public ngOnInit(): void {
         this.route.queryParams.subscribe((queryParams) => {
-            this.success = /true/.test(queryParams.success.toLowerCase());
+            this.success = queryParams.success.toLowerCase().includes('true');
```
Comment (Zabavsky) [inline src/app/account/social/social-register.component.ts]: !!! https://caniuse.com/#search=includes

File: src/app/admin/form-playground/form-playground.component.ts [hunk 2]
```diff
@@ -298,6 +298,6 @@ export class FormPlaygroundComponent implements OnInit {
 
     private _filterSpec(value: string): XmEntitySpec[] {
         return this.specs
-            .filter((option) => option.key.toLowerCase().indexOf(value.toLowerCase()) === 0);
+            .filter((option) => option.key.toLowerCase().startsWith(value.toLowerCase()));
```
Comment (Zabavsky) [inline src/app/admin/form-playground/form-playground.component.ts]: !!! https://caniuse.com/#search=startsWith

File: src/polyfills.ts [hunk 3]
```diff
@@ -64,6 +64,7 @@ import 'zone.js/dist/zone';  // Included with Angular CLI.
  */
 // import 'intl';  // Run `npm install --save intl`.
 
+// eslint-disable-next-line @typescript-eslint/unbound-method
 if (!String.prototype.startsWith) {
```
Comment (Zabavsky) [inline src/polyfills.ts]: nice 👍

File: src/app/account/social/social-register.component.ts (hunk 1 above)
Comment (Kaskyi) [reply src/app/account/social/social-register.component.ts]: For IE11 we are using `import 'core-js/es6/string';`, 
I've added a rule [prefer-string-starts-ends-with](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-string-starts-ends-with.md) - to sync the ways of codding.

File: src/polyfills.ts (hunk 3 above)
Comment (Kaskyi) [reply src/polyfills.ts]: Ill remove in the next iteration

File: src/app/admin/form-playground/form-playground.component.ts (hunk 2 above)
Comment (Kaskyi) [reply src/app/admin/form-playground/form-playground.component.ts]: The same reason `prefer-string-starts-ends-with`

**Pull request merged**